conu/backend/docker/image.py

```diff
@@ -153,13 +155,36 @@ def mount(self, mount_point=None):
         """
         return DockerImageFS(self, mount_point=mount_point)
 
+    def _run_container(self, run_command_instance, callback):
+        """ this is internal method """
+        tmpdir = mkdtemp()
+        tmpfile_fd, tmpfile = mkstemp(dir=tmpdir)
+        # the cid file must not exist
+        os.unlink(tmpfile)
```

Comment:
The file doesn't exists in newly created tmp directory anyway. Could we create only one temporary directory (value in backend or global), that is here created only if it doesn't exist previously? (one directory for all containers in session)

Reply:
good idea, I'm on it

Reply:
Fuf, this was way difficult than I'm willing to admit.